demo/src/player/player-dialog.js

```diff
@@ -41,6 +41,10 @@ export const openPlayerModal = ({ src, type, keySystems }) => {
   const player = Pillarbox.getPlayer('player');
 
   if (player.currentSrc() !== src) {
+    player.reset();
+    player.error(null);
+    player.titleBar.update({ title: undefined, description: undefined });
```

Comment:
In my opinion adding a function to the player that does this would be helpful for integrators.

Reply:
@jboix, this is role of the `reset` function, this is why I've done a few fixes to add the missing parts, see https://github.com/videojs/video.js/pull/8481, https://github.com/videojs/video.js/pull/8482

However, I should have commented that `player.error` and `player.titleBar` are temporary